general-module/build.gradle

```
@@ -130,4 +140,10 @@ uploadArchives {
     }
 }
 
+// コンパイル時に使用する文字コードをUTF-8で固定する
+tasks.withType(AbstractCompile).each { it.options.encoding = 'UTF-8' }
+tasks.withType(Test) {
+    systemProperty "file.encoding", "UTF-8"
+}
 // endregion maven repositories
+apply from: './jacoco.gradle'
```

143 ~ 147 行目の内容が jacoco の内容であれば、
region で囲って記述お願いします。

記述例
```suggestion
// endregion maven repositories

// region jacoco (code coverage option)

// コンパイル時に使用する文字コードをUTF-8で固定する
tasks.withType(AbstractCompile).each { it.options.encoding = 'UTF-8' }
tasks.withType(Test) {
    systemProperty "file.encoding", "UTF-8"
}

apply from: './jacoco.gradle' 

// endregion jacoco (code coverage option)
```